### PR TITLE
Register jay.is-a.dev

### PR DESCRIPTION
**Pull request by Jay-Dodhiawala**
Register jay.is-a.dev with CNAME record pointing to Aspiring Developer.

### CODE REVIEW

File: domains/jay.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Jay-Dodhiawala",
+           "email": "dodhiajk@myumanitoba.ca",
+           "discord": "619218152068481025"
+        },
+    
+        "record": {
+            "CNAME": "aspiring developer"
```
Comment (DEV-DIBSTER) [inline domains/jay.json:9]: Invalid CNAME record.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Review by omsenjalia — CHANGES_REQUESTED** [review]
Invalid CNAME

File: domains/jay.json (hunk 1 above)
Comment (Jay-Dodhiawala) [reply domains/jay.json:9]: what it should be

**Review by omsenjalia — APPROVED** [review]
ye this works

File: domains/jay.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "Jay-Dodhiawala",
+    "email": "dodhiajk@myumanitoba.ca",
+    "discord": "619218152068481025"
+  },
+
+  "record": {
+    "CNAME": "jaydodhiawala.github.io"
```
Comment (CuteDog5695) [inline domains/jay.json:9]: site gives 404 error

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**